src/Http/Authentication.Core/src/AuthenticationMetrics.cs

```diff
@@ -47,7 +47,7 @@ public AuthenticationMetrics(IMeterFactory meterFactory)
         _signOutCount = _meter.CreateCounter<long>(
             "aspnetcore.authentication.sign_outs",
             unit: "{request}",
-            description: "The total number of times a scheme is signed out.");
+            description: "The total number of times a principal is signed out.");
```

Comment:
@MackinnonBuck Can you look at this?

I assume "scheme" is right because we record the "aspnetcore.authentication.scheme" tag, but there's no per-user tag as that'd create too much cardinality.

Reply:
```suggestion
            description: "The total number of times a principal is signed out with a scheme.");
```